components/brave_shields/common/features.cc

```diff
@@ -47,5 +47,8 @@ const base::Feature kBraveDarkModeBlock{"BraveDarkModeBlock",
 // load the cosmetic filter rules using sync ipc
 const base::Feature kCosmeticFilteringSyncLoad{
     "CosmeticFilterSyncLoad", base::FEATURE_ENABLED_BY_DEFAULT};
+// When enabled, a mojom version of the panel will render
```

Comment:
nit: Might be more correct to say a "webui" version of the panel. It just uses mojom